torch/backends/_nnapi/serializer.py

```diff
@@ -452,6 +464,39 @@ def get_constant_value(self, jitval, typekind=None):
                 f"Expected constant value of type {typekind}, but got {ctype.kind()} for value '{jitval!r}'")
         return record
 
+    def transpose_to_nhwc(self, in_id, oper):
+        if oper.shape[2:] != (1, 1):
+            raise Exception("Automatic transpose only supported for H,W == 1,1")
+
+        out_oper = oper._replace(dim_order=DimOrder.CHANNELS_LAST)
+
+        inputs = [None] * 2
```

Comment:
any reason to create a `[None, None]` list and then replacing values vs just creating `inputs = [in_id, self.add_immediate_int_vector([0, 2, 3, 1])`?

(same with outputs)